model/src/main/kotlin/config/FileListStorageConfiguration.kt

```diff
@@ -30,32 +30,32 @@ import org.ossreviewtoolkit.utils.ort.storage.FileStorage
 import org.ossreviewtoolkit.utils.ort.storage.LocalFileStorage
 
 private const val TABLE_NAME = "provenance_file_listings"
```

Comment:
@fviernau, was it intentional that the table name continued to be named `provenance_file_listings` instead of `provenance_file_lists`?

Can we expect that this table name stays as-is now?

Reply:
In my merged renaming PR I just missed aligning the table name. Shall I file a PR quickly to fix it up ?